### PR TITLE
Feature regions

### PR DESCRIPTION
**Pull request by donnaaboise**
Include regions in to core ForestClaw routines

### CODE REVIEW

**Comment by cburstedde** [comment]
This looks like a truly massive update! :)

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -121,7 +122,7 @@ install(FILES
 	fclaw_pointer_map.h
 	fclaw_filesystem.h
 	fclaw_options.h
-	fclaw_gauges.h
```
Comment (scottaiton) [inline src/CMakeLists.txt:124]: fclaw_gauges.h shouldn't have been removed from installed headers in CMake

File: src/patches/clawpatch/CMakeLists.txt
Comment (scottaiton) [inline src/patches/clawpatch/CMakeLists.txt:1]: fclaw_clawpatch_regions.h should be added to the list of installed headers in clawpatch CMake

File: src/solvers/fc2d_geoclaw/CMakeLists.txt
Comment (scottaiton) [inline src/solvers/fc2d_geoclaw/CMakeLists.txt:1]: Shouldn't the Makefile.am be updated for geoclaw also?

File: applications/clawpack/shallow/2d/sphere/Makefile.am
Comment (scottaiton) [inline applications/clawpack/shallow/2d/sphere/Makefile.am:1]: the executable gets renamed to sphere_swe in autotools. This should probably be updated in CMake also

**Review by scottaiton — COMMENTED** [review]
Looked over the changes and the only issues I found is some build system discrepancies.

**Pull request merged**